contracts/samples/SimpleAccount.sol

```diff
@@ -97,7 +60,7 @@ contract SimpleAccount is BaseAccount, TokenCallbackHandler, UUPSUpgradeable, In
     }
 
     // Require the function call went through EntryPoint or owner
-    function _requireFromEntryPointOrOwner() internal view {
+    function _requireForExecute() internal view override virtual {
```

Comment:
This doesn't look like a good rename to me. Also, `virtual`? When do you expect the subclasses will need to override this function?

Reply:
it has different implementations for account with owner and 7702

Reply:
suggest a better name: it is "entryPointOrSelf" for 7702, and "entryPointOrOwner" for any other account (e.g. SimpleAccount), so neither name is good as "base name"

Reply:
Why is EntryPointOrOwner not good? Self can be seen as owner technically. Plus, requireAccessControl or requireAccessPermitted are better ifbyou want to understand what they are for